### PR TITLE
CGPROD 3121 confirm backgrounds

### PR DESCRIPTION
**Pull request by FostUK**
* Refactors
* Added icon Background
* Set panel background to fixed image
* Fix aspect ratio
* Lock height

### CODE REVIEW

File: src/components/game.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class Game extends Screen {
         console.log("Item used:", data); // eslint-disable-line no-console
         console.log("Item timer duration in seconds: ", data.duration || "No duration"); // eslint-disable-line no-console
         let time = data.duration * 1000;
-        let timedEvent = this.time.delayedCall(time, this.onEvent, [], this);
+        this.time.delayedCall(time, this.onEvent, [], this);
```
Comment (AnthonyGoodchild) [inline src/components/game.js:18]: odd how this wasn't linted

File: src/components/shop/confirm/confirm-resize.js [hunk 2]
```diff
@@ -3,8 +3,7 @@
  * @author BBC Children's D+E
  * @license Apache-2.0
  */
-import { getInnerRectBounds, getSafeArea } from "../shop-layout.js";
```
Comment (AnthonyGoodchild) [inline src/components/shop/confirm/confirm-resize.js:6]: I wonder if we should remove `getSafeArea` from `shop-layout.js`, should always be able to use `scene.layout.getSafeArea`

Comment (FostUK) [reply src/components/shop/confirm/confirm-resize.js:6]: Yeah I've been doing that where I see it. Just obscures what's going on.

File: src/components/shop/shop-menu-screen.js [hunk 3]
```diff
@@ -26,28 +26,24 @@ export class ShopMenu extends Screen {
         this.transientData.shop = { config: this.config.shopConfig };
         setBalance(this);
 
-        this.menu = createMenu(this);
+        const resize = createMenu(this);
```
Comment (AnthonyGoodchild) [inline src/components/shop/shop-menu-screen.js:29]: I don't like either way of doing this, it reads strange. Not sure how better to do it though..

File: src/main.js [hunk 4]
```diff
@@ -53,7 +53,7 @@ const screens = {
             debug: "debug",
             //Example of custom routing function
             next: scene => {
-                scene.navigate("narrative");
+                scene.navigate("shop-menu");
```
Comment (AnthonyGoodchild) [inline src/main.js:56]: 😂

File: src/components/game.js (hunk 1 above)
Comment (FostUK) [reply src/components/game.js:18]: Shows up on mine but as a warning.

File: src/components/shop/shop-menu-screen.js (hunk 3 above)
Comment (FostUK) [reply src/components/shop/shop-menu-screen.js:29]: Yeah - might talk about that in the dev channel - it actually seems to happen a lot - twice in this project.

I agree - feels dirty.

File: src/main.js (hunk 4 above)
Comment (FostUK) [reply src/main.js:56]: oh!!! :) Nice spot ta!

File: src/components/game.js (hunk 1 above)
Comment (AnthonyGoodchild) [reply src/components/game.js:18]: ah, yeah

**Review by AnthonyGoodchild — APPROVED** [review]
super fantastic ✌️

File: src/main.js (hunk 4 above)
Comment (AnthonyGoodchild) [reply src/main.js:56]: I keep doing this

**Pull request merged**